artemis-cli/src/main/java/org/apache/activemq/artemis/cli/process/ProcessBuilder.java

```diff
@@ -64,7 +65,7 @@ public static void cleanupProcess() {
     * @return
     * @throws Exception
     */
-   public static Process build(String logname, File location, boolean hook, String... args) throws Exception {
+   public static Process build(String logname, File location, boolean hook, String[] args, String[] properties) throws Exception {
```

Comment:
How do you use this?

Reply:
I mean that also: how that translate to the CLI help stuff? any way we can add some information how it's used?

Reply:
You can see how it's used in the Camel example that I added. See the camel-broker/pom.xml:
```
<properties>
   <!-- define the "war" property so it can be used in bootstrap.xml -->
   <param>war=camel-war-${project.version}.war</param>
</properties>
```
Keep in mind, this is a change used by the Maven plugin as it is the only user of org.apache.activemq.artemis.cli.process.ProcessBuilder#build.  There's no help information for the plugin that I'm aware of.

Reply:
From the cli itself.. say I wanted to start a server with -D<xmlconfigproperty=something>


it would be nice to have a way to define those properties from artemis run as well